nvml/pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = [
+    "hatchling>=0.12.0",
+    "setuptools<61",
```

Comment:
IIRC, we bumped the `setuptools` version in other integrations because of a CVE. Could you bump it to the version we now use? 

To make it easier, you could run `ddev create Integration` command which is going to create a new integration called `Integration` with all its project files, so you'll be able to easily compare what you have here and what is now generated by `ddev` for new integrations

Reply:
For sure - I've gone and updated the following things in this file (based on newly generated integration template):
```
    "hatchling>=0.13.0",
    "setuptools>=66; python_version > '3.0'"
    "setuptools; python_version < '3.0'"
    "datadog-checks-base>=25.1.0"
```